README.md

```
@@ -1,127 +1,162 @@
 <div align="center">
 
-<img src="assets/logo.svg" alt="VEET — Universal Linux App Uninstaller" width="600"/>
+<img src="assets/icons/icon.png" alt="VEET Logo" width="130" height="130"/>
 
-<br/>
+# VEET
+
+### Universal Linux Application Uninstaller &amp; Deep-Clean Residual Purger
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)